packages/e2e-tests/src/specs/dashboard/deleteStory.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  publishStory,
+  visitDashboard,
+  insertStoryTitle,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Stories Dashboard', () => {
+  it('should delete story', async () => {
+    //need to create new story as all stories are deleted before test runs.
+    await createNewStory();
+    await insertStoryTitle('Delete this story');
+    await publishStory();
+    await visitDashboard();
+
+    await expect(page).toMatchElement('h2', { text: 'Dashboard' });
+    await page.hover('div[data-testid^=story-grid-item]');
```

Comment:
Maybe selecting based on the `aria-label` is a bit more specific / self-documenting? Since there could be multiple grid items on the page.

`page.hover('[role="listitem"][aria-label="Details about Delete this story"]')`

WDYT?

Reply:
@swissspidy  yes we could do that too, but i dont think there will be multiple grid items on the page since before every test all the stories are deleted. Still would make this change to make it simple